src/common/Questionnaire/Questionnaire.jsx

```diff
@@ -64,27 +64,30 @@ const Questionnaire = ({ update, onUpdate, questions, answers, setAnswers }) =>
   );
 
   return (
-    <DashboardCard className={classes.root}>
-      <Toolbar className={classes.toolbar}>
-        {update ? renderEditButtons() : <div />}
-        <div>
-          <Typography className={classes.questionsHeader}>
-            {t('interviewDashboard.questionnaire.questions')}
-            <span>{` (${t('unitControlPage.totalText')} ${questions.length})`}</span>
-          </Typography>
-        </div>
-      </Toolbar>
-      <Container>
-        <List
-          component='nav'
-          className={classes.list}
-        >
-          {questions.length ? questions.map((question, index) => (
+    <ResizableBox
+      light
+      height={300}
+      width={1000}
+      minConstraints={[300, 150]}
+      content={(
+        <DashboardCard className={classes.root}>
+          <Toolbar className={classes.toolbar}>
+            {update ? renderEditButtons() : <div />}
```

Comment:
Change to fragment instead of useless div
```suggestion
            {update ? renderEditButtons() : <></>}
```

Reply:
I want it to be an empty div (if you run it you can tell it is moving the div aside and I don't want it)